crates/svm-builds/build.rs

```diff
@@ -110,7 +121,16 @@ pub const TARGET_PLATFORM: &str = "{}";
 
 fn generate() {
     let platform = get_platform();
-    let releases = svm::blocking_all_releases(platform).expect("Failed to fetch releases");
+
+    let releases: Releases = if let Ok(file_path) = std::env::var(SVM_RELEASES_LIST_JSON) {
+        let file = File::open(file_path)
+            .expect("SVM_RELEASES_LIST_JSON defined, but cannot read the file referenced");
```

Comment:
I would recommend replacing the string here with the const from above (just in case a user actually encounters this issue, but in the meantime the value above in the source code was updated for some reason).

So:
```suggestion
            .expect(&format!("{:?} defined, but cannot read the file referenced", SVM_RELEASES_LIST_JSON));
```